include/itkVariationalRegistrationElasticRegularizer.hxx

```diff
@@ -345,9 +345,9 @@ VariationalRegistrationElasticRegularizer< TDisplacementField >
 ::SolveElasticLESThreaderCallback( void * arg )
 {
   //Get MultiThreader struct
-  auto* threadStruct = (MultiThreader::ThreadInfoStruct *) arg;
-  int threadId = threadStruct->ThreadID;
-  int threadCount = threadStruct->NumberOfThreads;
+  auto* threadStruct = (MultiThreaderBase::WorkUnitInfo *) arg;
```

Comment:
If these are using the classing threading backend, should they be calling `this->DynamicMultiThreadingOff()`.

Reply:
This class uses `GenerateData`, which calls `SolveElasticLES`, which sets up multi-threading.